target/linux/ramips/dts/mt7621_beeline_smartbox-turbo.dts

```diff
@@ -87,7 +37,7 @@
 		#size-cells = <1>;
 
 		partition@0 {
-			label = "u-boot";
```

Comment:
I would like you not to change this generally accepted bootloader label.

Reply:
Specified a label as in the OEM firmware. Isn't that preferable?

Reply:
There are two different aspects here:
1. In the commit "Add support for Device 1" you're trying to fix Device 2. I don't think this is a good idea. Would be better to fix another devices in the separate commits with description why do we need these fixes.
1. `u-boot` is a standard well known name for bootloader partition in OpenWrt. Like `rootfs`. So, I would prefer to keep this label.